perfkitbenchmarker/providers/openstack/os_virtual_machine.py

```diff
@@ -221,12 +213,28 @@ def _CheckFloatingIPNetworkExists(self):
   def _CheckNetworkExists(self):
     cmd = os_utils.OpenStackCLICommand(self, 'network', 'show',
                                        self.network_name)
+    self._IssueCommandCheck(cmd, 'Network', self.network_name)
+
+  def _IssueCommandCheck(self, cmd, name, id):
```

Comment:
use a different name for the `id` parameter since it overshadows the `id()` function.